IBAnimatable/TransitionAnimationType.swift

```diff
@@ -23,6 +24,8 @@ public enum TransitionAnimationType {
   static func fromString(transitionType: String) -> TransitionAnimationType? {
     if transitionType.hasPrefix("Fade") {
       return fadeTransitionAnmationType(transitionType)
+    } else if transitionType.hasPrefix("SystemRippleEffect") {
```

Comment:
Since the enum isn't a string, we can't make constructors as suggested. I agree that would vibe really nicer. Any suggestions?